boards/arm/thingy53_nrf5340/thingy53_nrf5340_common.dtsi

```diff
@@ -167,6 +167,17 @@
 	status = "okay";
 };
 
+/* StemmaQT/Qwiic/Groove external connector */
+&i2c0 {
+	compatible = "nordic,nrf-twim";
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_FAST>;
```

Comment:
One more thought, would it make sense to leave the bus to standard bitrate? Just in case someone connects a low speed device and wants to use it with no overlay. Unless there's some mention about 400khz support on the stemma spec of course.